wav_in

```diff
@@ -5,3 +5,31 @@
   dtmin = 50.
   initfile = "./input/restart.ww3"
 /
+&input_nml
+  input%forcing%water_levels = 'T'
+  input%forcing%currents     = 'C'
+  input%forcing%winds        = 'C'
+  input%forcing%ice_conc     = 'C'
+  input%forcing%ice_param1   = 'C'
+  input%forcing%ice_param5   = 'C'
+/
+&output_type_nml
+  type%field%list = 'WND ICE HS T02 T0M1 T01 FP DIR EF USS TOC IC1 IC5'
+  type%point%file = 'ww3_points.list'
+/
+&output_date_nml
+  date%field%outffile  = '1'
+  date%field%start     = '19990101 0'
+  date%field%stride    = '86400'
+  date%field%stop      = '99990101 0'
```

Comment:
And this be 2019? (Although maybe it will cycle around, so leaving the default is better?)

Reply:
Changed in this commit e8fd445